packages/remix-dev/config/routes.ts

```diff
@@ -54,6 +54,13 @@ export interface DefineRouteOptions {
    * Should be `true` if this is an index route that does not allow child routes.
    */
   index?: boolean;
+
+  /**
+   * Should be a optional unique id string for this route, named like its `file`
+   * but without the extension. Use this optional param if you need to aggregate two
+   * or more routes to the same route file.
```

Comment:
The description confused me a little, so I had a go at rewriting it. The "named like its `file` but without the extension" part was the main bit that I didn't understand. Is that just a suggestion for what to use as an ID?
```suggestion
   * An optional unique id string for this route, appended to the file name e.g:
   * `<file-name>--<id>`. Use this if you need to aggregate two or more routes
   * with the same route file.
```

Reply:
Hi @AndrewLeedham 

I agree, the original comment with "extension" term are confusing, because de `unique ID` can be any term that are really unique and didn't conflict with the "default" one.